doctrine/doctrine-bundle/1.6/manifest.json

```diff
@@ -8,10 +8,10 @@
     },
     "env": {
         "#1": "Format described at https://www.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/configuration.html#connecting-using-a-url",
-        "#2": "For an SQLite database, use: \"sqlite:///%kernel.project_dir%/var/data.db\"",
-        "#3": "For a MySQL database, use: \"mysql://db_user:db_password@127.0.0.1:3306/db_name?serverVersion=5.7\"",
-        "#4": "IMPORTANT: You MUST configure your server version, either here or in config/packages/doctrine.yaml",
-        "DATABASE_URL": "postgresql://db_user:db_password@127.0.0.1:5432/db_name?serverVersion=13&charset=utf8"
+        "#2": "IMPORTANT: You MUST configure your server version, either here or in config/packages/doctrine.yaml",
+        "#3": "DATABASE_URL=\"sqlite:///%kernel.project_dir%/var/data.db\"",
+        "#4": "DATABASE_URL=\"mysql://db_user:db_password@127.0.0.1:3306/db_name?serverVersion=5.7\"",
+        "DATABASE_URL": "postgresql://db_user:db_password@127.0.0.1:5432/db_name?serverVersion=11&charset=utf8"
```

Comment:
Why using 11 here (vs 13 in the config)?

Reply:
I have no idea how I did that!